Sources/UnleashProxyClientSwift/UnleashProxyClientSwift.swift

```diff
@@ -98,6 +124,12 @@ public class UnleashClientBase {
         SwiftEventBus.unregister(self, name: name)
     }
 
+    public func updateContext(_ newContext: Context) -> Void {
```

Comment:
Similar to the new `init` I found this method is useful for integrating, I have more/full control over the `Context` values, whereas with the other `updateContext` method there are issues like if you don't set existing context values for `userId`, etc then they will be cleared.